From: Remi Collet <remi@remirepo.net>
Date: Thu, 26 Sep 2024 15:49:03 +0200
Subject: adapt GHSA-9pqp-7h25-4f32 test for 7.x

(cherry picked from commit 29065f33f37f99ba33254cb23c941647bcd7372c)
(cherry picked from commit 87ed9429a17e38daec4dcfd7a3c3db194197ccb3)
(cherry picked from commit d97de82afe8696b6d76cc11bc7b6d6c2652d06d9)
(cherry picked from commit 64a9dfdec2cb530428c9cbe90f98f346c5d23797)
(cherry picked from commit fb9688a470070f0ab656e2f94efbee2988a30eaf)
(cherry picked from commit 355cd5cb0c3813ae49d88693cff410281413c2f6)
---
 tests/basic/GHSA-9pqp-7h25-4f32.phpt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tests/basic/GHSA-9pqp-7h25-4f32.phpt b/tests/basic/GHSA-9pqp-7h25-4f32.phpt
index 29bcb65..b913edc 100644
--- a/tests/basic/GHSA-9pqp-7h25-4f32.phpt
+++ b/tests/basic/GHSA-9pqp-7h25-4f32.phpt
@@ -21,8 +21,10 @@ function test($boundaryLen) {
         getenv('TEST_PHP_CGI_EXECUTABLE'),
         '-C',
         '-n',
+        '-dlog_errors=1',
         __DIR__ . '/GHSA-9pqp-7h25-4f32.inc',
     ];
+    $cmd = implode(' ', $cmd);
 
     $boundary = str_repeat('A', $boundaryLen);
     $body = ""
@@ -92,11 +94,10 @@ array(1) {
 
 Boundary len: 5117
 Starting...
+PHP Warning:  Boundary too large in multipart/form-data POST data in Unknown on line 0
 X-Powered-By: %s
 Content-type: text/html; charset=UTF-8
 
-<br />
-<b>Warning</b>:  Boundary too large in multipart/form-data POST data in <b>Unknown</b> on line <b>0</b><br />
 Hello world
 array(0) {
 }
